package.json

```diff
@@ -5,24 +5,18 @@
   "license": "MIT",
```

Comment:
Maybe save this again, to remove the esy specific formatting?

Reply:
What do you mean?

Reply:
OK got it. need to run prettier :0